pkg/serverless/trace/inferredspan/span_enrichment_test.go

```diff
@@ -21,6 +22,79 @@ const (
 	dataFile = "../testdata/event_samples/"
 )
 
+// TestGetServiceMapping checks if the function correctly parses the input string into a map.
+func TestCreateServiceMapping(t *testing.T) {
+	// Case 1: Normal case
```

Comment:
can we add empty string case and ",a:b," case?

Reply:
added cases